gensrc/script/vectorized/vectorized_functions.py

```diff
@@ -530,7 +530,7 @@
     [100014, 'last_query_id', 'VARCHAR', [], "UtilityFunctions::last_query_id"],
     [100015, 'uuid', 'VARCHAR', [], "UtilityFunctions::uuid"],
     [100016, 'uuid_numeric', 'LARGEINT', [], "UtilityFunctions::uuid_numeric"],
-    [100017, 'assert_true', 'BOOLEAN', ['BOOLEAN'], 'UtilityFunctions::assert_true'],
+    [100017, 'assert_true', 'BOOLEAN', ['BOOLEAN', "VARCHAR"], 'UtilityFunctions::assert_true'],
```

Comment:
You need to add a new function rather than changing the old function's definition
When cluster is upgrading, new backends will execute as there would be two inputs, but it is only one. Because this is generated by the old FE.

Reply:
Done